### PR TITLE
Databases week1

### PR DESCRIPTION
**Pull request by maribelpreite**

### CODE REVIEW

File: databases/week1/homework_queries.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+FROM task;
+
+SELECT COUNT(*)
+AS tasks_without_due_date
+FROM task
+WHERE due_date IS NULL;
+
+SELECT *
+FROM task
+WHERE status_id = (SELECT id FROM status WHERE name = 'Done');
```
Comment (NatsGt) [inline databases/week1/homework_queries.sql]: I Like how you practice using an inner query. But I thnk a Join would be better here.

File: databases/week1/homework_queries.sql [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+SELECT *
+FROM task
+WHERE status_id != (SELECT id FROM status WHERE name = 'Done');
```
Comment (NatsGt) [inline databases/week1/homework_queries.sql]: Same as above

**Review by NatsGt — APPROVED** [review]
Very good job. Just so you know, each time you need an Inner Join, you can just write JOIN, the default for SQL is inner join anyway.

File: databases/week1/homework_queries.sql (hunk 1 above)
Comment (maribelpreite) [reply databases/week1/homework_queries.sql:12]: Thank you! I've changed inner join to just join!

Comment (maribelpreite) [reply databases/week1/homework_queries.sql]: You're right! Thank you for your suggestion! I've changed it now :)

**Pull request merged**